js/main.js

```diff
@@ -36,5 +44,69 @@ const getRandomInt = (min,max) => {
 function checkLength(string, maxLength) {
   return string.length < maxLength;
 }
+//создает url строку
+function createUrl() {
+  return `photos/${getRandomPositiveInteger(1,25)}.jpg`;
+}
+
+function createAvatar() {
+  return `img/avatar-${getRandomPositiveInteger(1,6)}.svg`;
+}
+
+//массив для descripton
+const variantsOfDescriptions = ['живописно','удивительно','красивый вид','любимое фото','дежавю'];
+
+const variantsOfMessages = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+
+const variantsOfNames = [
+  'Артём',
+  'Алиса',
+  'Женя',
+  'Саша',
+  'Коля'
+];
+
+const createName = () => variantsOfNames[getRandomPositiveInteger(0,4)];
+
+const createMessage = () => variantsOfMessages[getRandomPositiveInteger(0,5)];
+
+const createDescription = () => variantsOfDescriptions[getRandomPositiveInteger(0,4)];
+
+function createFormObject() {
+  return ({
+    id: getRandomPositiveInteger(1,25),
+    url: createUrl(),
+    description: createDescription,
```

Comment:
```suggestion
    description: createDescription(),
```
функцию надо вызывать